addons/l10n_jo_edi/models/res_company.py

```diff
@@ -0,0 +1,14 @@
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    l10n_jo_edi_sequence_income_source = fields.Char(string="Sequence of Income Source")
+    l10n_jo_edi_secret_key = fields.Char(string="Jordan EINV Secret Key")
+    l10n_jo_edi_client_id = fields.Char(string="Jordan EINV Client ID")
+    l10n_jo_edi_taxpayer_type = fields.Selection(selection=[
+        ('income', "Unregistered in the sales tax"),
+        ('sales', "Registered in the sales tax"),
```

Comment:
So only when it is sales will you have refunds?

Reply:
No, in all taxpayer types we can have refunds.

Reply:
What is the _is_sales_refund method needed for then?